src/Computation/IConstraints.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Symbolica.Computation;
 
-internal interface IModel : IDisposable
+internal interface IConstraints : IDisposable
 {
     bool IsSatisfiable(IValue assertion);
```

Comment:
`AreSatisfiable`?

Reply:
It's asking if the assertion passed in is satisfiable within the constraints, so I think it is fine.